flags/eigendaflags/cli.go

```diff
@@ -46,6 +49,26 @@ func CLIFlags(envPrefix, category string) []cli.Flag {
 			EnvVars:  []string{withEnvPrefix(envPrefix, "DISPERSER_RPC")},
 			Category: category,
 		},
+		&cli.DurationFlag{
+			Name:     ResponseTimeoutFlagName,
+			Usage:    "Flag used to configure the underlying disperser-client. Total time to wait for the disperseBlob call to return or disperseAuthenticatedBlob stream to finish and close.",
+			Value:    60 * time.Second,
+			EnvVars:  []string{withEnvPrefix(envPrefix, "RESPONSE_TIMEOUT")},
+			Category: category,
+		},
+		&cli.DurationFlag{
+			Name: ConfirmationTimeoutFlagName,
+			Usage: `The total amount of time that the client will spend waiting for EigenDA
+			to "confirm" (include onchain) a blob after it has been dispersed. Note that
+			we stick to "confirm" here but this really means InclusionTimeout,
+			not confirmation in the sense of confirmation depth.
+			
+			If ConfirmationTimeout time passes and the blob is not yet confirmed,
+			the client will return an api.ErrorFailover to let the caller failover to EthDA.`,
```

Comment:
Nit - we use three sentences here to describe a single variable's usage. maybe better to condense this for less user cognitive complexity

Reply:
I think all three sentences say different things and are important though? I'm personally a fan of having too much information - find it much more useful than not enough.